### ISSUE TITLE
Todo for history page

### ISSUE THREAD

**Issue by jospoortvliet** [owncloud/owncloud.org#696]
- [x] add pictures of offices to history page
- [x] the 1.2 release should come after the developer meetup, as it was released at the end of it.

Comment or edit to add more.

**Comment by jospoortvliet** [comment]
@jancborchardt @cosenal @karlitschek

**Comment by jancborchardt** [comment]
Scrolling through it from top to bottom, here’s what I found: ;)

* [ ] add conference logos for example to the FOSDEM event
* [ ] better picture for Nuremberg office @dragotin @jnweiger maybe you have one?
* [ ] fix image of new Stuttgart office cc @PVince81 @nickvergessen @schiesbn take a nice pic!
* [ ] add Red Hat logo to the partnering up event
* [ ] »ownCloud gets Community Manager« … come on :D I know you’re great, but that doesn’t belong there as a big event
* [ ] picture for February 2013 dev meeting. No picture @karlitschek?
* [ ] picture for August 2012 meeting. Here as well @karlitschek? Also the link is broken @jospoortvliet 
* [ ] original Android app launch date missing
* [ ] add screenshot to »ownCloud 3 launched« entry
* [ ] add ownCloud Inc logo to »ownCloud, Inc. announced« entry
* [ ] picture for October 2011 dev meeting in Zürich … maybe anyone has something @karlitschek @FreeMinded @blizzz @icewind1991 @jakobsack?
* [ ] add website logos of the press coverage to »First major press coverage« entry

TL;DR more images & logos to make it more visual!

**Comment by nickvergessen** [comment]
![img_20150924_083655](https://cloud.githubusercontent.com/assets/213943/10066952/1a41a626-6298-11e5-93a8-c68a4007d13f.jpg)

**Comment by jancborchardt** [comment]
@nickvergessen cmon! At least try a bit and put some people in there. :D @schiesbn @PVince81 @nickvergessen sit in your chairs and smile and get a photo taken by some of the other people there in the office.

**Comment by schiesbn** [comment]
That's a photo from the day we moved to the new office.

![20140815_120433](https://cloud.githubusercontent.com/assets/1589737/10073490/958728d4-62c9-11e5-8a9b-cbeb89cd5ff4.jpg)

**Comment by jancborchardt** [comment]
Photo quality is less good with the lighting, but at least it’s got some people in it :+1: ;D – try to make an even better one soon, it’s on our history page!

**Comment by FreeMinded** [comment]
@jancborchardt unfortunately I have no pictures from the Zurich Meet-Up.

**Comment by jospoortvliet** [comment]
* No idea of the original android app launch date. Give me a blog or link...
* no 'ownCloud 3 launched' screenshots that I could find, otherwise they would be there...
* There's a video for FOSDEM... now you want multiple pictures for each event? Dude...

Added a few other things:
* 'owncloud picks up smashbox from cern' July 2015
* 'ownCloud visit to CERN' November 2014
* March 2014 Inc funding round
* user numbers to end of year statistics

So otherwise - I just need pictures and screenshots for the other events you mention and I will be happy to add them.

**Comment by jospoortvliet** [comment]
closing. More wishes can be added anytime but generally, we're good.
